Getting-Started/Code/Creating-Forms/index.md

```diff
@@ -102,7 +102,7 @@ In the partial view we're going to render our contact form using the view model
 
 
 #### Adding the macro
-Last thing to do before we can add the form to a page is to allow the macro in a richtext editor.
+Last thing to do before we can add the form to a page is to allow the macro in a rich text editor.
```

Comment:
I'm not sure this reads correctly, starting a sentence with 'Last thing'... also the following sentence has a typo on to/two...

... how about this instead?

The last thing to do before we can add the form to a page is to 'allow the Macro to be used in a rich text editor'. Select and expand the **Macros** node and find the 'Contact Form' Macro, (created automatically when we saved our partial view with 'Create Macro' checked). 

Under the Editor Settings, tick the two checkboxes:

- Use in rich text editor and the grid
- Render in rich text editor and the grid

Now, on a page containing a rich text editor, you can add the form to the page via the _Add Macro_ button.